Resources/Locale/en-US/advertisements/vending/sovietsoda.ftl

```diff
@@ -0,0 +1,5 @@
+﻿advertisement-sovietsoda-1 = For Tsar and Country.
```

Comment:
i feel like soviet soda wouldn't like the Tsar

Reply:
Yeah I honestly don't know who came up with that, can you come up with something better perhaps?

Reply:
For Comrade and Country?
For Comrade and Party?

Reply:
Changed to "For comrade and country."